build/azDevOps/azure/api-contract-tests.yml

```diff
@@ -0,0 +1,75 @@
+name: $(build.SourceBranchName)-init
+
+# A pipeline with no trigger
+trigger: none
+
+resources:
+  repositories:
+    - repository: templates
+      type: github
+      name: amido/stacks-pipeline-templates
+      # Created when you set up the connection to GitHub from Azure DevOps
+      endpoint: amidostacks
+
+variables:
+  - group: amido-stacks-infra-credentials-nonprod
+
+  # Versioning
+  - name: Version.Major
+    value: 1
+  - name: Version.Minor
+    value: 2
+  - name: Version.MajorMinor
+    value: $(Version.Major).$(Version.Minor)
+  - name: Version.Revision
+    value: $[counter(variables['Version.MajorMinor'], 0)]
+
+  # Tests
+  - name: Test.ProviderContractTestProjectSourcePath
+    value: '$(Build.SourcesDirectory)/src/api/xxAMIDOxx.xxSTACKSxx.API.ContractTests'
+
+  # Docker Config
+  - name: Docker.ImageTag
+    value: $(Version.MajorMinor).$(Version.Revision)-$(build.sourcebranchname)
+
+  # Pact Broker Info
+  - name: Test.PactBrokerUrl
+    value: https://amido-stacks.pactflow.io
+  # Specific Contract Test Variable Declarations
+  - name: BROKER_URL
+    value: $(Test.PactBrokerUrl)
+  - name: BUILD_NUMBER
+    value: $(Docker.ImageTag)
+  - name: PACT_BEARER_TOKEN
+    value: $(Test.PactBearerToken)
+
+stages:
+  - stage: Build
+    jobs:
+      # Set build number
+      - template: azDevOps/azure/templates/jobs/build-updatebuildnumber.yml@templates
+        parameters:
+          build_number: $(Docker.ImageTag)
+
+      - job: Build
+        dependsOn: UpdateBuildNumber
+        pool:
+          vmImage: 'ubuntu-latest'
+        continueOnError: False
+        steps:
+
+          - task: UseDotNet@2
+            displayName: 'Use .NET Core SDK 6.0.x'
+            inputs:
+              packageType: sdk
+              version: "6.0.x"
+              installationPath: $(Agent.ToolsDirectory)/dotnet
```

Comment:
Isn't this project dotnet 8 now?

Reply:
Good spot. This pipeline isn't running currently